overlord/ifacestate/ifacemgr.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/interfaces/backends"
 	"github.com/snapcore/snapd/overlord/hookstate"
+	"github.com/snapcore/snapd/overlord/ifacestate/ifacerepo"
```

Comment:
not sure about "ifacerepo" name,  I suppose "repo" alone would have been ok, it depends also what kind of helpers it will grow later